content/en/installing-pixie/04-install-schemes/02-yaml.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "YAML"
-metaTitle: "Install | Install Schemes | YAML"
+metaTitle: "Install | Install Schemes (optional) | YAML"
 metaDescription: "How to install Pixie via YAML"
 order: 2
```

Comment:
This file needs a first step added:

1. Install the Pixie CLI

The CLI is used to get Pixie's YAML files. You can install the Pixie CLI following the directions [here](/installing-pixie/install-schemes/cli/).